### DISCUSSION TITLE
Failed to connect: couldn't dial whatsapp web websocket

### DISCUSSION THREAD

**Discussion by xFJournet** [tulir/whatsmeow#245] [WhatsApp protocol Q&A]
[Main ERROR] Failed to connect: couldn't dial whatsapp web websocket: dial tcp 31.13.67.33:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

**Comment by spi433d** [comment]
you may use this version it will solve all this [na3 whatsapp](https://gbshine.com/na3-whatsapp-apk) it will connect to the host in a smoothly way

**Comment by panzhi1** [comment]
can you solved?
